hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/security/acl/OzoneNativeAuthorizer.java

```diff
@@ -92,12 +91,24 @@ public boolean checkAccess(IOzoneObj ozObject, RequestContext context)
     }
 
     // bypass all checks for admin
-    boolean isAdmin = isAdmin(context.getClientUgi());
+    boolean isAdmin = isAdmin(ozAdmins, context.getClientUgi());
     if (isAdmin) {
       return true;
     }
 
     boolean isOwner = isOwner(context.getClientUgi(), context.getOwnerName());
+
+    boolean isReadOnlyAdmin = isAdmin(ozReadOnlyAdmins,
+        context.getClientUgi());
+
+    // bypass read checks for read only admin users
+    if (isReadOnlyAdmin
+        && (context.getAclRights() == ACLType.READ
+        || context.getAclRights() == ACLType.READ_ACL
+        || context.getAclRights() == ACLType.LIST)) {
+      return true;
+    }
```

Comment:
We can put the check before the isOwner statement.

Reply:
@z-bb , thanks for updating the patch. It looks overall good. Just this minor comment. 

Since you have created the sub jira for this feature, shall we update the title of this patch to "HDDS-8455. Om supports read only administrators." ?

Reply:
> We can put the check before the isOwner statement.

@ChenSammi done



> @z-bb , thanks for updating the patch. It looks overall good. Just this minor comment.
> 
> Since you have created the sub jira for this feature, shall we update the title of this patch to "[HDDS-8455](https://issues.apache.org/jira/browse/HDDS-8455). Om supports read only administrators." ?

@ChenSammi done